deploy/kustomize/patches/driverinfo-ephemeral-kubernetes-1-16-patch.yaml

```diff
@@ -0,0 +1,6 @@
+# Append volumeLifeCycltModes to DervierInfo valid only for Kubernetes v1.16+
```

Comment:
Typos.

Reply:
> Instead of copying quite a few files into `deploy/kustomize/kubernetes-1.16-*` dirs, would it be possible to take the existing Kubernetes 1.15 deployment and just patch the `volumeLifecycleField` in?

Do you mean something this: https://github.com/avalluri/pmem-CSI/commit/099d91ddd135232b12608ee31cf79640edf3bec1

Reply:
> Typos.

Fixed